Description: Fixes a compilation error (maybe a difference between javac and ecj?)
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: no
--- a/bundles/org.eclipse.team.ui/src/org/eclipse/team/internal/ui/mapping/ModelEnablementPreferencePage.java
+++ b/bundles/org.eclipse.team.ui/src/org/eclipse/team/internal/ui/mapping/ModelEnablementPreferencePage.java
@@ -179,7 +179,7 @@
 	public boolean performOk() {
 		Object[] checked = tableViewer.getCheckedElements();
 		Set<ITeamContentProviderDescriptor> nowEnabled = new HashSet<>();
-		nowEnabled.addAll((Collection<? extends ITeamContentProviderDescriptor>) Arrays.asList(checked));
+		nowEnabled.addAll((Collection) Arrays.asList(checked));
 		if (hasDescriptorEnablementChanged(checked)) {
 			TeamUI.getTeamContentProviderManager().setEnabledDescriptors(
 					nowEnabled.toArray(new ITeamContentProviderDescriptor[nowEnabled.size()]));
